cmd/utils/remoteConfig.go

```diff
@@ -0,0 +1,127 @@
+/*
+* Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package utils
+
+import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+)
+
+var RemoteConfigData RemoteConfig
+
+func init() {
+
+	InitRemoteConfigData()
+}
+
+func (remoteConfig *RemoteConfig) AddRemote(name string, host string, port string) error {
+
+	remotes := &RemoteConfigData.Remotes
+	if _, exists := (*remotes)[name]; exists {
+		return errors.New("remote already added")
+	}
+
+	remote := Remote{Url: host, Port: port}
+	(*remotes)[name] = remote
+
+	return nil
+}
+
+func (remoteConfig *RemoteConfig) UpdateRemote(name string, host string, port string) error {
+
+	remotes := &RemoteConfigData.Remotes
+	if _, exists := (*remotes)[name]; !exists {
+		return errors.New("no such remote")
+	}
```

Comment:
Found a 4 line (31 tokens) duplication in the following files: 
Starting at line 49 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
Starting at line 65 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
Starting at line 81 of /home/asanka/repos/wso2/micro-integrator/cmd/utils/remoteConfig.go
```go
func (remoteConfig *RemoteConfig) AddRemote(name string, host string, port string) error {

	remotes := &RemoteConfigData.Remotes
	if _, exists := (*remotes)[name]; exists {
```

Reply:
As discussed offline,
It is not worth moving two lines to a separate method and calling it.